src/v2/Apps/Artists/ArtistsApp.tsx

```diff
@@ -0,0 +1,270 @@
+
+                    <FollowArtistButtonQueryRenderer
+                      id={link.internalID}
+                      contextModule={ContextModule.featuredArtistsRail}
+                    />
+                  </Box>
+                </RouterLink>
+              )
+            })}
+          </Carousel>
+
+          {genes.map(gene => {
+            if (gene.trendingArtists.length === 0) return null
+
+            return (
+              <Box key={gene.name} my={3}>
+                <Text as="h2" variant="subtitle" my={1}>
+                  {gene.name}
+                </Text>
+
+                <CSSGrid
+                  gridColumnGap={1}
+                  gridRowGap={1}
+                  gridTemplateColumns={[
+                    "repeat(1, 1fr)",
+                    "repeat(2, 1fr)",
+                    "repeat(2, 1fr)",
+                    "repeat(4, 1fr)",
+                  ]}
+                >
+                  {gene.trendingArtists.map(artist => {
+                    const { image } = artist
+
+                    if (!image) return null
+
+                    return (
+                      <Box
+                        key={artist.internalID}
+                        display="flex"
+                        flexDirection="column"
+                        justifyContent="space-between"
+                        height="100%"
+                        p={1}
+                        border="1px solid"
+                        borderColor="black10"
+                        flex="1"
+                      >
+                        <RouterLink
+                          to={artist.href}
+                          noUnderline
+                          style={{ display: "block" }}
+                        >
+                          <ResponsiveBox
+                            aspectWidth={image.thumb.width}
+                            aspectHeight={image.thumb.height}
+                            maxWidth="100%"
+                          >
+                            <Image
+                              src={image.thumb.src}
+                              srcSet={image.thumb.srcSet}
+                              alt={artist.name}
+                              width="100%"
+                              height="100%"
+                              lazyLoad
+                            />
+                          </ResponsiveBox>
+
+                          <Box my={1}>
+                            <Text variant="mediumText">{artist.name}</Text>
+
+                            {artist.formattedNationalityAndBirthday && (
+                              <Text color="black60">
+                                {artist.formattedNationalityAndBirthday}
+                              </Text>
+                            )}
+
+                            <Text color="black60">
+                              {artist.counts.artworks} works,{" "}
+                              {artist.counts.forSaleArtworks} for sale
```

Comment:
A couple display rules that exist on the old surface which we might want to port over:
- Pluralization of "work/works" (probably rare that we only have 1 artwork for a featured gene though)
- If `counts.artworks === counts.forSaleArtworks`, omit `, N for sale`

https://github.com/artsy/force/blob/master/src/desktop/components/artist_cell/index.jade#L16